src/Request/AbstractRequest.php

```diff
@@ -10,7 +12,6 @@
 use ChromaX\BasicHttpClient\Request\Transport\HttpsTransport;
 use ChromaX\BasicHttpClient\Request\Transport\HttpTransport;
 use ChromaX\BasicHttpClient\Response\ResponseInterface;
-use ChromaX\CommonException\NetworkException\Base\NetworkException;
```

Comment:
Is it correct that the NetworkException is not in use anymore?

Reply:
Yes, it was used only in docs